### PR TITLE
Using jackson instead of fastjson.

### PR DESCRIPTION
**Pull request by QilongZhang**

### CODE REVIEW

**Comment by QilongZhang** [comment]
#94

File: healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/AfterHealthCheckCallbackProcessor.java [hunk 1]
```diff
@@ -128,7 +128,7 @@ private boolean doApplicationAfterHealthCheckCallback() {
                     logger.error("SOFABoot application after readiness check callback("
                                  + afterReadinessCheckCallback.getClass()
                                  + ") failed, the details is: "
-                                 + JSON.toJSONString(health.getDetails()));
+                                 + new ObjectMapper().writeValueAsString(health.getDetails()));
```
Comment (ujjboy) [inline healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/AfterHealthCheckCallbackProcessor.java]: ObjectMapper can reuse, try singleton.

File: healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/AfterHealthCheckCallbackProcessor.java [hunk 2]
```diff
@@ -77,7 +77,7 @@ private boolean doMiddlewareAfterHealthCheckCallback() {
                     logger.error("SOFABoot middleware after readiness check callback("
                                  + middlewareAfterReadinessCheckCallback.getClass()
                                  + ") failed, the details is: "
-                                 + JSON.toJSONString(health.getDetails()));
+                                 + new ObjectMapper().writeValueAsString(health.getDetails()));
```
Comment (ujjboy) [inline healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/AfterHealthCheckCallbackProcessor.java]: ObjectMapper can reuse, try singleton.

File: healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/AfterHealthCheckCallbackProcessor.java (hunk 1 above)
Comment (QilongZhang) [reply healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/AfterHealthCheckCallbackProcessor.java]: ok~

File: healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/HealthIndicatorCheckProcessor.java [hunk 3]
```diff
@@ -36,9 +36,11 @@
  * @version $Id: HealthIndicatorCheckProcessor.java, v 0.1 2017年10月20日 上午10:55 liangen Exp $
  */
 public class HealthIndicatorCheckProcessor {
-    private static Logger logger = SofaBootHealthCheckLoggerFactory
-                                     .getLogger(HealthIndicatorCheckProcessor.class
-                                         .getCanonicalName());
+    private static Logger             logger       = SofaBootHealthCheckLoggerFactory
+                                                       .getLogger(HealthIndicatorCheckProcessor.class
+                                                           .getCanonicalName());
```
Comment (ujjboy) [inline healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/HealthIndicatorCheckProcessor.java]: Change to class.

Comment (QilongZhang) [reply healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/HealthIndicatorCheckProcessor.java]: @ujjboy  done~

**Pull request merged**